packages/eslint-config-globis/shared.js

```diff
@@ -9,6 +9,18 @@ module.exports = {
   parser: '@typescript-eslint/parser',
   plugins: ['prettier', '@typescript-eslint'],
   rules: {
+    // FixMe https://github.com/benmosher/eslint-plugin-import/issues/1615
+    'import/extensions': [
+      'error',
+      'ignorePackages',
+      {
+        js: 'never',
+        jsx: 'never',
+        mjs: 'never',
+        ts: 'never',
+        tsx: 'never',
+      },
+    ],
```

Comment:
What is difference from `"import/extensions": ["error", "never"]` ?

Reply:
I wanted to set is "do not set the extension of JavaScript files".
Also, I want to enter `.svg` without omitting.

Reply:
ah hah 👍